src/Altinn.App.Core/Features/Correspondence/Models/CorrespondenceNotificationOrderResponse.cs

```diff
@@ -0,0 +1,75 @@
+using System.Text.Json.Serialization;
+
+namespace Altinn.App.Core.Features.Correspondence.Models;
+
+/// <summary>
+/// Represents a notification connected to a specific correspondence
+/// </summary>
+public sealed record CorrespondenceNotificationOrderResponse
+{
+    /// <summary>
+    /// The id of the notification order
+    /// </summary>
+    [JsonPropertyName("id")]
+    public required string Id { get; set; }
+
+    /// <summary>
+    /// The senders reference of the notification
+    /// </summary>
+    [JsonPropertyName("sendersReference")]
+    public string? SendersReference { get; set; }
+
+    /// <summary>
+    /// The requested send time of the notification
+    /// </summary>
+    [JsonPropertyName("requestedSendTime")]
+    public DateTime RequestedSendTime { get; set; }
```

Comment:
Can we be consistent in the use of datetime types? I believe when serialized we end up with the same format regardless

Reply:
These types come from the Correspondence repo itself. It's possible that we can change them all to DateTimeOffset types, I haven't actually attempted that.

Reply:
I hope that's possible, since in JSON this will be ISO8601 regardless of type